### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by ChrisTregaskis**
This is my header section and Nav bar.
Nav bar is not supposed to be sticky.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+<html lang="en-gb">
+
+    <head>
+        
+        <title>Chris Tregaskis Portfolio!</title>
+        
+        <!--==== LINKS ====-->
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="stylesheet" type="text/css" href="queries.css">
+        <link rel="stylesheet" type="text/css" href="normalize.css">
```
Comment (markandmarkup) [inline index.html]: You want to put normalize above the other two links - remember CSS uses the last style it's presented with!

File: queries.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        width: 50%;
+    }     
+}
+
+
+/*--- MOBILE MAX-WIDTH 480 ----*/
+
+@media screen (max-width: 480px) {
+    nav a {
+
```
Comment (markandmarkup) [inline queries.css]: You have a selector here with nothing in it. It it's not used it can be removed.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+/*-----------------------------------*/
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html,
+body {
```
Comment (markandmarkup) [inline style.css]: Do you need to use both html and body tags here? It seems like one or the other would be enough

File: style.css [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    position: absolute;
+    width: 100%;
+    bottom: 2%;
+    margin-left: 8%;
+    color: #f7f9fb;
+    overflow: auto;
+}
+
+.profile-pic {
+    background-image: url(imgs/chris-headshot.jpg);
```
Comment (markandmarkup) [inline style.css]: Image is not loading - doesn't appear to be in your repo

File: index.html [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+        <link href="https://fonts.googleapis.com/css?family=Muli:300,400,400i,700&display=swap" rel="stylesheet">
+    
+    </head>
+
+    <body>
+        
+        <header> 
+            <!--==== NAV ====-->
+            <nav>
```
Comment (markandmarkup) [inline index.html]: Your nav is very hard to see - seems like a white on white background

File: index.html [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+                <summary>
+                    <h3>Chris Tregaskis</h3>
+                    <h4>Full Stack Developer</h4>
+                    <div class="summary-social">
+                        <button>Get In Touch</button>
+                        <ul>
+                            <li><a href="#"><ion-icon name="logo-github"></ion-icon></a></li>
+                            <li><a href="#"><ion-icon name="logo-codepen"></ion-icon></a></li>
+                            <li><a href="#"><ion-icon name="logo-instagram"></ion-icon></a></li>
+                            <li><a href="#"><ion-icon name="logo-linkedin"></ion-icon></a></li>
```
Comment (markandmarkup) [inline index.html]: None of these icons appear to be loading. Are they external links?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+                        <li><a href="#">portfolio</a></li>
+                        <li><a href="#">contact</a></li>
+                    </ul>
+                </div>
+            </nav>
+            <!--==== HEADER PORTFOLIO INFO ====-->
+            <div class="row header-info">
+                <!-- img in css -->
+                <div class="profile-pic"></div>
+                <summary>
```
Comment (markandmarkup) [inline index.html]: The summary box is appearing with a small scroll bar in chrome

File: queries.css [hunk 8]
```diff
@@ -0,0 +1,31 @@
+/*---- TABLETS MAX-WIDTH 768 ----*/
+
+@media screen and (max-width: 768px) {
+    
+    summary ul {
+        visibility: hidden;
+    }
+    
+    summary button {
+        width: 50%;
```
Comment (markandmarkup) [inline queries.css]: There's a moment close to the breakpoint when this button freaks out a bit

File: queries.css [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+    }     
+}
+
+
+/*--- MOBILE MAX-WIDTH 480 ----*/
```
Comment (markandmarkup) [inline queries.css]: Squashing the window really small causes some overlaps in your nav links

File: index.html [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+                        <li><a href="#"><ion-icon name="logo-codepen"></ion-icon></a></li>
+                        <li><a href="#"><ion-icon name="logo-instagram"></ion-icon></a></li>
+                        <li><a href="#"><ion-icon name="logo-linkedin"></ion-icon></a></li>
+                    </ul>
+                </div>
+            </div>
+        
+        </footer>
+        
+-->
```
Comment (markandmarkup) [inline index.html]: Charlie mentioned to me earlier that it's not good practice to include commented out code in your merges. Might want to check this with him.

File: index.html (hunk 6 above)
Comment (ChrisTregaskis) [reply index.html]: thanks! All links working.. totally forgot to add the links

File: index.html (hunk 10 above)
Comment (ChrisTregaskis) [reply index.html]: took out the footer and cleared out commented out code any, thanks!

File: style.css (hunk 3 above)
Comment (ChrisTregaskis) [reply style.css]: good point!

**Pull request merged**